modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -1423,6 +1426,97 @@ neo4j_home$ --nodes persons.csv --nodes games.csv --id-type string
 The `id` property of the nodes in the `persons` group will be stored as `long` type, while the `id` property of the nodes in the `games` group will be stored as `string` type, as the global `id-type` is a string.
 ====
 
+[role=label--new-2025.01.0 label--block-only]
+== Applying changes to data via CSV files
+
+You can use CSV files to update your data during incremental import.
+
+This feature is especially useful when you want to update existing nodes, relationships, labels, or properties in large datasets.
+To be able to use this feature, each entity in the dataset must be associated with a unique ID.
+
+[NOTE]
+====
+This feature is supported only by `block` format.
+====
+
+=== Set an explicit action for each row
+
+You can set an explicit action for each row in the CSV file by using the `:ACTION` keyword in the header file.
+If no action is specified, the import tool works as in full import mode, creating new data.
+
+The following actions are supported:
+
+* `empty` = `CREATE` (default)
+* `C`, `CREATE` - Creates new nodes and relationships with or without properties.
+* `U`, `UPDATE` - Updates existing nodes, relationships, or properties.
+Leaving the property identifier empty is like having a “*” wild card.
+* `D`, `DELETE` - Deletes an existing node or relationship.
+Deleting a node also deletes its relationships (`DETACH DELETE`).
+
+
+.Using actions in CSV files to update data
+[source, cypher, role="nocopy"]
+----
+:ACTION,personId:ID,name,:LABEL
+CREATE,1,"Keanu Reeves",Actor
+UPDATE,2,"Laurence Fishburne",Actor
+DELETE,4
+----
+
+=== Update existing labels
+
+You can add or remove one or more labels from an existing node by prepending the clause `LABEL` in the header with a a `+` (default) or `-`:
+
+* `:+LABEL` - Add one or more labels to an existing node.
+* `:-LABEL` - Remove one or more labels (if they exist) from an existing node.
+
+Multiple labels are separated by a semicolon.
+
+For example, a file could have the following format:
+
+[source, csv]
+----
+p0:ID,:+LABEL,:-LABEL,name,age
+person1,Actor,Person,"Keanu Reeves",55
+person2,Actor;Director,Person,"Laurence Fishburne",60
+person3,Actor,Person,"Carrie-Anne Moss",53
+----
+
+In this case, all labels in the second column are added and all the labels in the third column are removed (if they exist).
+The nodes are identified by their `p0` key.
+
+=== Update existing properties
+
+You can add or remove properties from an existing entity by prepending the clause `PROPERTY` in the header with a `+` or `-`:
+
+* `:+PROPERTY` - Add one or more properties (if they exist) to an existing entity.
```

Comment:
This isn't how it works. `:+PROPERTY` doesn't exist. You specify properties using their name in the column header.